src/app/editor/providers/editor-effects.service.ts

```diff
@@ -69,7 +69,7 @@ export class EditorEffectsService {
             language: language,
         };
 
-        return this.api.project.createNode({ project: projectName }, nodeCreateRequest)
+        return this.api.project.createNode({ project: projectName, lang: language } as any, nodeCreateRequest)
```

Comment:
Add a comment explaining the extra lang param, with a link to https://github.com/gentics/mesh/issues/404